typescript/ws-protocol/src/FoxgloveServer.test.ts

```diff
@@ -280,14 +279,9 @@ describe("FoxgloveServer", () => {
         "message",
         {
           channel: { id: 42, topic: "foo", encoding: "json", schemaName: "baz" },
-          data: new DataView(expectedPayload.buffer),
+          data: expectedPayload,
         },
       ]);
-      const msg = msgEvent[1] as ClientPublish;
-      expect(msg.data.byteLength).toEqual(expectedPayload.byteLength);
-      for (let i = 0; i < expectedPayload.byteLength; i++) {
-        expect(msg.data.getUint8(i)).toEqual(expectedPayload[i]);
-      }
```

Comment:
wonder why this was here? was jest not able to compare DataView by default?

Reply:
It was able to compare DataView. No idea why it was here in the first place